plc4j/drivers/s7/src/main/java/org/apache/plc4x/java/s7/readwrite/protocol/S7ProtocolLogic.java

```diff
@@ -136,32 +154,27 @@ public void onConnect(ConversationContext<TPKTPacket> context) {
                                 return;
                             }
 
-                            // Prepare a message to request the remote to identify itself.
-                            LOGGER.debug("Sending S7 Identification Request");
-                            TPKTPacket tpktPacket = createIdentifyRemoteMessage();
-                            context.sendRequest(tpktPacket)
-                                .expectResponse(TPKTPacket.class, REQUEST_TIMEOUT)
-                                .check(p -> p.getPayload() instanceof COTPPacketData)
-                                .unwrap(p -> ((COTPPacketData) p.getPayload()))
-                                .check(p -> p.getPayload() instanceof S7MessageUserData)
-                                .unwrap(p -> ((S7MessageUserData) p.getPayload()))
-                                .check(p -> p.getPayload() instanceof S7PayloadUserData)
-                                .handle(messageUserData -> {
-                                    LOGGER.debug("Got S7 Identification Response");
-                                    S7PayloadUserData payloadUserData = (S7PayloadUserData) messageUserData.getPayload();
-                                    extractControllerTypeAndFireConnected(context, payloadUserData);
-                                });
-                        });
-                });
-        }
-        // This usually when we're running a passive mode river.
-        else {
-            LOGGER.info("S7 Driver running in PASSIVE mode.");
-            s7DriverContext.setPassiveMode(true);
-
-            // No login required, just confirm that we're connected.
-            context.fireConnected();
-        }
```

Comment:
We should keep this as we are planning on making the drivers passive capable.

Reply:
It is moved up in the method body so we do not have bigger block with active connection under if brackets.